theme/scripts/index.ts

```diff
@@ -1 +1,17 @@
 import "../styles/index.scss";
+const navOpenMenuButton = document.getElementById('nav-open-menu-button');
```

Comment:
Would be cool to make this a separate module that we import/initialize in index.ts. This will get a bit hectic when we have more js/ts to write.